### DISCUSSION TITLE
CONFIG_I2C_SLAVE - dropped?

### DISCUSSION THREAD

**Discussion by gkujawsk** [zephyrproject-rtos/zephyr#49892] [Q&A] [answered]
Has support for CONFIG_I2C_SLAVE has been dropped silently in 3.1?
Before 3.1 code compiled w/o any issues. Is there a migration guide or the i2c slave mode support is totally removed?

**Comment by yashi** [comment, marked as answer]
Hmm..  [The release note](https://docs.zephyrproject.org/3.1.0/releases/release-notes-3.1.html) doesn't mention it. But "[i2c: Update terminology to match i2c spec](https://github.com/zephyrproject-rtos/zephyr/pull/43166)" is merged for v3.1.

**Comment by gkujawsk** [comment]
CONFIG_I2C_SLAVE changed to CONFIG_I2C_TARGET and all the i2c_slave_xxx function changed to i2c_target_xxx
@yashi thank you so much for pointing this out.
